src/Notifications.svelte

```diff
@@ -104,7 +104,15 @@
 		default: '#aaaaaa'
   }
 
-  export let timeout = 3000
+  export let options = {
+      timeout: 3000,
+      width: '40vw'
+  }
+
+  onMount(() => {
+      const toasts = document.querySelector('.toasts');
```

Comment:
Probably shouldn't use querySelector in Svelte - just:

`bind:this={toasts}` on the toasts element, and then remove this line.

Reply:
I removed the querySelector. I also included changes so that not every property needs to be specified in options. 

Also, CSS properties are supported in every browser except IE.